rms/external_services/tests/test_client.py

```diff
@@ -0,0 +1,55 @@
+from fastapi.testclient import TestClient
+import pytest
+from fastapi import FastAPI
+from ..views import router
+from ..models import SearchBody
+
+app = FastAPI()
+app.include_router(router, prefix="/external_services")
+
+
+@pytest.fixture
+def test_client():
+    return TestClient(app)
+
+
+def test_search_scopus(test_client):
```

Comment:
As discussed please mock the outbound api calls

Reply:
fixed